lib/board.rb

```diff
@@ -0,0 +1,32 @@
+require "pry"
```

Comment:
Should probably move this into the spec_helper, so it isn't loaded when running the app "for real."

Reply:
💡